### PR TITLE
README: Add well-known branches

### PR DESCRIPTION
**Pull request by lemoer**

### CODE REVIEW

**Comment by lemoer** [comment]
@1977er As you recently mentioned, that you did not know which branch is suited for what... Is this sufficient for you?

**Comment by 1977er** [comment]
Guess so.

**Review by 1977er — APPROVED** [review]

**Review by Dark4MD — APPROVED** [review]

**Review by s-2 — APPROVED** [review]
`next` may sometimes contain patches introducing support for devices not yet in OpenWrt master (though maybe these should go into a different branch instead?)

**Comment by lemoer** [comment]
@s-2 In general, @AiyionPrime and I discussed that it would be maybe a good idea to use feature branches for purposes, that are not covered by this well-known branches. These branches could be more volatile. We also discussed that we would like those feature branches to be cleaned up a bit (maybe squashed or so) if they are merged to well-known branches (if they are merged at all). Does this suit your needs?

**Comment by s-2** [comment]
I agree `next` as a well-known branch should probably reflect upstream gluon-next directly, without any further commits from our side, i.e. experimental device support could go into separate feature branches.

Currently, it's quite a hassle to build combinations of features (e.g. there is only fastd for devices from the next branch), not sure how to address this without creating an even more confusing mess of additional branches though.

// edit: maybe we can have something like "multiple inheritance", i.e. automated cherry-picking from different branches at build time?

**Comment by AiyionPrime** [comment]
Looks good.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# build a current ffh gluon
+## well-known branches
+
+|    Branch Name   |       Description      |                Builds Against the Following Gluon Version                |            Used For           |
```
Comment (AiyionPrime) [inline README.md:3]: I find it irritating to have the entry `manual builds` in every row;
but that's just DBS calling "Normalize the database", I'm afraid.
Fine with me.

**Review by AiyionPrime — APPROVED** [review]

**Comment by lemoer** [comment]
@s-2 Atm, automating this seems to be an overkill to me. Maybe you can hack some shell scripts for your purpose to track two feature branches easily (or so).

**Pull request merged**